solidity/truffle-config.js

```diff
@@ -48,7 +48,7 @@ module.exports = {
 
     alfajores: {
       provider: function () {
-        const kit = Kit.newKit("https://alfajores-forno.celo-testnet.org")
+        const kit = Kit.newKit(process.env.CELO_HOSTNAME)
```

Comment:
Maybe we should expect the same-named env variables for all networks.

So instead of having `ETH_HOSTNAME` and `CELO_HOSTNAME` we could expect `CHAIN_API_URL`,
and instead of `CONTRACT_OWNER_ETH_ACCOUNT_PRIVATE_KEY` and `CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY` we could expect `CONTRACT_OWNER_ACCOUNT_PRIVATE_KEY`.

This will make the migrations more generic, hence from the CI workflows perspective, we could handle both Ethereum and Celo migrations in one step.

Reply:
I did unified the variables (see 76adcf0cbc3f47ab59cdc1bb3d774799de51705c), but I'm not sure if it will allow us to handle both migrations in one step (see my comment below).

Reply:
I was also thinking about loading secrets for the environemnt based on the environment property.  So we would name the GH environments the same as we expect workflow dispatch `environment` property to be set.

Reply:
The downside of this solution would be the need to repeat the same settings across different repositories. Sticking with current solution seems to be the least complicated.